DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -43,6 +43,7 @@ public function load(array $configs, ContainerBuilder $container)
         $config = $this->processConfiguration($configuration, $configs);
 
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('matcher.xml');
```

Comment:
I now simply always load `matcher.xml`, which means `rule_matcher` and `request_matcher` will be loaded even if they are not used. I don’t find this a big problem. We could make their loading optional, for instance in the `createRule|RequestMatcher()` methods.